dev/test/document.ts

```diff
@@ -913,6 +913,8 @@ describe('set document', () => {
         requestEquals(
           request,
           set({
+            document: document('documentId'),
+            mask: updateMask(),
```

Comment:
This will break the conformance tests. We should also not set an empty update mask.

Reply:
removed here and in other unit tests that used an empty update mask